src/routes/transactionWrapper.js

```diff
@@ -1,13 +1,28 @@
 import { sequelize } from '../models';
-import audit from '../models/auditModelGenerator';
+import handleErrors from '../lib/apiErrorHandler';
+
+const namespace = 'SERVICE:WRAPPER';
+
+const logContext = {
+  namespace,
+};
 
 export default function transactionWrapper(originalFunction) {
   return async function wrapper(req, res, next) {
-    return sequelize.transaction(async () => {
-      await audit.addAuditTransactionSettings(sequelize, null, null, 'transaction', originalFunction.name);
-      const result = await originalFunction(req, res, next);
-      audit.removeFromAuditedTransactions();
-      return result;
-    });
+    let error;
+    try {
+      return sequelize.transaction(async () => {
+        let result;
+        try {
+          result = await originalFunction(req, res, next);
+        } catch (err) {
+          error = err;
+          throw err;
```

Comment:
This will not crash the server, wouldn't the inner try/catch simply pass the caught error to the outer catch without providing any other functionality?  (the transaction gets rolled back regardless)

Reply:
correct, but it does save off the original error so it can be correctly logged and not just logging a failed transaction.

Reply:
We really don't need it, since the err would be caught by the outer catch. It would make more sense if there was some other logic in the inner catch, e.g. a different logging message. 
 Since we need this PR quickly, I am fine with leaving it and refactoring later.

Reply:
I think the idea here is that the error coming out of the transaction failure wouldn't necessarily look the same as the error coming out of the function that caused the error within the transaction.

Reply:
Perhaps given the confusion it would be nice to add a comment explaining the intent